### PR TITLE
Add `videoCodec` option

### LINKED ISSUE #45: Ability to specify codec
[AVFoundation allows specifying the codec used in a recording](https://developer.apple.com/documentation/avfoundation/avvideocodectype). It would be very nice to support this in Aperture so that it can be included in Kap's settings.

### PR DESCRIPTION
**Pull request by sindresorhus**
Fixes #45

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ test('returns audio sources', async t => {
   }
 });
 
+test('returns available video codecs', t => {
+  const codecs = aperture.videoCodecs;
+  console.log('Video codecs:', codecs);
```
Comment (vadimdemedes) [inline test.js]: Is console.log intentionally left here?

Comment (sindresorhus) [reply test.js]: Yes, just to manually verify everything is correct. I do it in the other test too.

Comment (vadimdemedes) [reply test.js]: Why not just `t.deepEqual()`? There's probably a reason for it.

Comment (sindresorhus) [reply test.js]: The contents are not static. I could do deepEqual with some conditionals, but then I'm just reproducing the main code. This is just easier.

Comment (vadimdemedes) [reply test.js]: 👍

**Pull request merged**